ptx/src/pass/test/insert_implicit_conversions/default_relaxed.ptx

```diff
@@ -0,0 +1,24 @@
+.version 6.5
+.target sm_30
+.address_size 64
+
+.func (.reg .b32 output1, .reg .b32 output2) default_relaxed(
+    .reg .b64 input
+)
+{
+    ld.b64   { output1, output2 }, [input];
+    ret;
+}
+
+// %%% output %%%
+
+.func (.reg .b32 %2, .reg .b32 %3) %1 (
+    .reg .b64 %4
+)
+{
+    .b64 %7 = zluda.convert_implicit.bit_to_ptr.b64.reg.b64 %4;
```

Comment:
I guess after the conversion ZLUDA tracks `%7` as a variable of type `.b64` in `.generic` space? In this case, would it be possible to have this line instead be as follows do readability? Just for variables that are in non-`.reg` address space.
`.generic.b64 %7 = zluda.convert_implicit.bit_to_ptr.b64.reg.b64 %4;`